src/base58.app.src

```diff
@@ -1,11 +1,13 @@
 {application, base58,
  [
+  {pkg_name, "ae_erl_base58"},
   {description, "A simple Erlang module to perform Base58 encoding and decoding"},
-  {vsn, "0.0.1"},
+  {vsn, "0.0.2"},
   {registered, []},
   {applications, [
                   kernel,
                   stdlib
                  ]},
-  {env, []}
+  {env, []},
+  {licenses,["MIT","ISC"]}
```

Comment:
Are those the correct licences? I see it is originally [under `apache2.0`](https://github.com/aeternity/erl-base58/blob/master/LICENCE)?